docs/config.json

```diff
@@ -99,7 +99,8 @@
             { "title": "Integrating with JWT", "slug": "/application-access/guides/jwt/" },
             { "title": "API Access", "slug": "/application-access/guides/api-access/" },
             { "title": "AWS Console Access", "slug": "/application-access/guides/aws-console/" },
-            { "title": "Dynamic Registration", "slug": "/application-access/guides/dynamic-registration/" }
+            { "title": "Dynamic Registration", "slug": "/application-access/guides/dynamic-registration/" },
+            { "title": "Automation", "slug": "/application-access/guides/automation/" }
```

Comment:
I wonder if something like `Service Accounts` would be a better name for this page, @klizhentas let me know what you think.

Reply:
I like the "service account" terminology, it is used in plenty of other software, e.g. K8s.

Reply:
I agree, service accounts is a well-understood term - better than robots